contributors/guide/owners.md

```diff
@@ -47,6 +51,8 @@ reviewers:
   - alice
   - carol   # this is another comment
   - sig-foo # this is an alias
+security:
+  - alice
```

Comment:
It's usually easy enough to track someone down via their github handle, but not always (email hidden, different slack handle). Do we want to require contact information here?

Reply:
oh good point we probably only want emails, but then idk if people want their email in a repo...

Reply:
We have had privacy/legal issues previously around putting personal information (such as e-mail addresses) inside repos. You need explicit consent from anyone who you do this for. Not impossible, but can be difficult to enforce and audit.

Reply:
IIRC, there were issues with requiring personal info about people be published on github

Reply:
https://github.com/kubernetes/test-infra/issues/3271#issuecomment-327607473